tests/test_bench.py

```diff
@@ -107,13 +107,13 @@ def test_load_from_json_stackstac(fake_dask_client, bench_site1, bench_site2):
         resampling="nearest",
         extra={
             "odc-stac": {"groupby": "solar_day", "stac_cfg": CFG},
-            "stackstac": {"dtype": "uint16", "fill_value": 0},
```

Comment:
can you explain why the change of dtype please?

Reply:
it significantly increases memory requirement (4 times)

Reply:
Hi @Kirill888 ,
It's because the uint16 values on stackstac don't support (anymore?) the `fill_value=0` , so the easiest way to deal with that was to change to `int64`.
```ValueError: The fill_value 0 is incompatible with the output dtype uint16. Either use `dtype='int64'`, or pick a different `fill_value`.```

Reply:
what version of `stackstac` and `numpy` is that? My main concern here is that this change puts `stackstac` at disadvantage. Also this error looks to me like a numpy 2 upgrade issue.